projects/syncdex-finance/index.js

```diff
@@ -0,0 +1,36 @@
+const sdk = require("@defillama/sdk");
+
+const staking_contract = "0xa8CD01322Ad632c9656879e99Fd7FbC11ca8E3BB";
+
+const assets = [
+  // other tokens which probably for some reason was sent to the contract accidentally
+  "0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4",
+  "0x3a34FA9a1288597Ad6C1Da709f001D37FeF8b19e",
```

Comment:
sorry, you can't include your projects own tokens as tvl

Reply:
> sorry, we cant list your project till there is some tvl
Hi team, the contract has just been added with some TVL and will be gradually migrated to it in the next few days. Please check it and list the projects to help Era's ecosystem grow stronger. Have a nice day team!